spec/features/airline_pagination_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature "paginating airlines" do
+  scenario "user visits index and does not see 26th airline" do
+    25.times do
+      FactoryGirl.create(:airline, name: Faker::Company.name)
+    end
+
+    FactoryGirl.create(:airline, name: "second_page_air")
+    visit airlines_path
+
+    expect(page).to_not have_content("second_page_air")
+  end
+
+  scenario "user clicks next to see 26th airline" do
+    25.times do
+      FactoryGirl.create(:airline, name: Faker::Company.name)
+    end
```

Comment:
same here, maybe factory girl create_list doesn't let override the name, but try first